pkg/apis/edge/v1alpha1/single-placement.go

```diff
@@ -51,17 +52,16 @@ type SinglePlacementSlice struct {
 
 // SinglePlacement describes one Location that matches the relevant EdgePlacement.
 type SinglePlacement struct {
-	// `location` refers to the relevant Location.
-	Location ExternalName `json:"location"`
+	// Cluster is the logicacluster.Name of the logical cluster that contains
+	// both the Location and the SyncTarget.
+	Cluster string `json:"cluster"`
+
```

Comment:
If Cluster contains the Location and the SyncTarget, can't we retrieve & save them using the Cluster reference? (instead of explicitly hold them in the SinglePlacement   ?

Reply:
What system component do you mean by "we"?

One logical cluster can contain several Location&SyncTarget pairs, so I do not see how it is unnecessary to identify them here.

Reply:
The _cluster_ holds the SyncTarget and Location objects; the _cluster name_ does _not_ hold the _names_ of the SyncTarget and Location objects.

Reply:
A client of a kcp server needs to provide a cluster name whenever accessing an API object.